src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -8,293 +8,491 @@
+                GT_Values.NI,
+                null,
+                210,
                 120);
+        // Lepidolite
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GlauconiteSand, 13L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Lepidolite, 20L),
                 ItemList.Cell_Empty.get(4L),
                 GT_Values.NF,
+                Materials.Fluorine.getGas(2000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Lithium, 3L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_Values.NI,
+                GT_Values.NI,
+                null,
+                260,
+                120);
+        // Glauconite Sand
+        GT_Values.RA.addElectrolyzerRecipe(
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GlauconiteSand, 21L),
+                ItemList.Cell_Empty.get(6L),
+                GT_Values.NF,
                 Materials.Hydrogen.getGas(2000L),
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 6L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
                 378,
                 120);
+        // Glauconite Dust
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glauconite, 13L),
-                ItemList.Cell_Empty.get(4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glauconite, 21L),
+                ItemList.Cell_Empty.get(6L),
                 GT_Values.NF,
                 Materials.Hydrogen.getGas(2000L),
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 6L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
                 378,
                 120);
+        // Vermiculite
```

Comment:
incorrect total, should just be 29. also ideally keep the water intact

Reply:
Same reason as above for water change, I'm also not sure here why the total is incorrect, it seems right? Give me your way of calculating so maybe I understand the problem better

Reply:
3 Fe + 4 Al + 4 Si + 2 H + 4 H2O + 12 O. so 29. I think you are confused over how compounds work in chem balance. though you did it correctly for red granite

Reply:
Same reason as above, quite funny

Reply:
29 Vermiculite dust -> 4 Water + 2H + 6O + 3 Fe + 10 alumina + 4 raw silicon

Reply:
It's quite weird that the recipe outputs water and hydrogen with oxygen, but ok I guess

Reply:
same as fullers earth and a few others